lib/screens/proposal/card.dart

```diff
@@ -80,13 +87,16 @@ class _ProposalCardState extends State<ProposalCard> {
         children: <Widget>[
           Padding(
             padding: const EdgeInsets.only(bottom: 8.0),
-            child: Text(
-              "${AppLocalizations.of(context).translate("numDaysUntilVotingEnds")} ${widget.proposal.date.difference(DateTime.now()).inDays.toString()}",
-              style: TextStyle(
-                fontSize: 15,
-                color: Colors.black,
-              ),
-            ),
+            child: widget.proposal.date.difference(DateTime.now()).inDays > 0
+                ? Text(
+                    // TODO: needa translate "Voting is over!"
+                    "${AppLocalizations.of(context).translate("numDaysUntilVotingEnds")} ${widget.proposal.date.difference(DateTime.now()).inDays.toString()}",
+                    style: TextStyle(
+                      fontSize: 15,
+                      color: Colors.black,
+                    ),
+                  )
+                : Text("Voting is over!"),
```

Comment:
check if this logic is valid and add in "Voting is over!" to static translations